services/backend/src/recommendation_system/recommendation_flow/model_prediction/SitongModel.py

```diff
@@ -0,0 +1,122 @@
+import random
```

Comment:
if this is someone's model, please still label it with Alpha first. Ideally in a folder named Alpha

Reply:
also is this file used anywhere?